ui-tests/src/test/resources/features/connections/connections.feature

```diff
@@ -129,7 +129,7 @@ Feature: Connections CRUD test
     And click on the "Create" button
     Then Camilla is presented with the Syndesis page "Connections"
 
-    When Camilla selects the "my sample sf conn" connection
+    When Camilla selects "my sample sf conn" connection type
```

Comment:
This step needs to be changed to that new `open detail`

Reply:
This step selects connection type/connector in connection-create process. It doesn't open connection detail

Reply:
Imho if the next steps is `Then Camilla is presented with "my sample sf conn" connection details` and this verify step hangs because of detail page isn't opened.

Reply:
This happens after connection-create process, it's done in previous block of steps.

Reply:
sorry, you're right

Reply:
fixed, pushed